### PR TITLE
Update to Gradle 4.10.2

### PR DESCRIPTION
**Pull request by chetanmeh**
Updates to version 4.10.2 (from 4.8.1). This brings some improvement in [maven publishing](https://docs.gradle.org/4.9/release-notes.html) which would be useful for us in publishing our artifacts

See release notes [here](https://gradle.org/releases/)

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [x] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by chetanmeh** [comment]
@vvraskin Can you trigger a PG run for this PR

**Comment by chetanmeh** [comment]
@csantanapr Can you do a PG run for this?

**Comment by csantanapr** [comment]
@chetanmeh Are all repos should be updated?
Or not necessary for now

**Comment by chetanmeh** [comment]
No change required in other repos

**Comment by rabbah** [comment]
@csantanapr is this good to merge? No issues for me locally.

**Comment by chetanmeh** [comment]
The build fails in install due to gradle/gradle#6849. There is a possible workaround

```
As a workaround, you can create a src/main/scala/ScalaForTestsWorkaround.scala with something like class ScalaForTestsWorkaround in it.
```

Now I see [Gradle 5 also released](https://docs.gradle.org/5.0/release-notes.html) on Nov 26. May be we switch to that as it shows good performance improvement (though referred bug still exist there)

**Comment by chetanmeh** [comment]
@csantanapr Can you try again now a PG run with updated PR?

File: tests/src/main/scala/org/apache/openwhisk/GradleWorkaround.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.openwhisk
+
+/**
+ *  This class is a workaround for https://github.com/gradle/gradle/issues/6849
+ */
+class GradleWorkaround {}
```
Comment (csantanapr) [inline tests/src/main/scala/org/apache/openwhisk/GradleWorkaround.scala]: 🤦‍♂️ 🤷‍♂️

**Comment by csantanapr** [comment]
PG looks good

**Pull request merged**